FirebaseDev.podspec

```diff
@@ -16,6 +16,7 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.10'
```

Comment:
Good point. The downside is that the FirebaseDev pod is incompatible with the Firebase pod and could cause confusion, especially for developers incrementally wanting to add iOS Firebase features not currently open sourced. I'll talk to the product team next week.